resources/metadata/kaniko.yaml

```diff
@@ -6,6 +6,7 @@ spec:
   inputs:
     secrets:
       - name: dockerConfigJsonCredentialsId
+        description: Jenkins 'Secret file' credentials ID containing Docker config.json (with registry credential(s)). You can create it like explained in the Docker Success Center in the article about [how to generate a new auth in the config.json file](https://success.docker.com/article/generate-new-auth-in-config-json-file).
```

Comment:
The [HowTo](https://success.docker.com/article/generate-new-auth-in-config-json-file) is not that helpful when working on a Mac as per default the Mac keychain is used to store credentials. I created a [GIST](https://gist.github.com/CCFenner/060ddb98c84463ff4dc101c364774237) where it is described how to create the `config.json` manually.